defaults/api_ents.go

```diff
@@ -0,0 +1,51 @@
+package defaults
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/docker/libentitlement/entitlement"
+	"github.com/docker/libentitlement/secprofile"
+)
+
+const (
+	engineDomain = "engine"
+
+	// FIXME: we need to define the possible domain, identifier and value for the entitlement.
+	APIEntID = engineDomain + ".api"
+	// e.g. defining an ent like so: "engine.api=swarm:all:allow"
+	APIEntAllowID = APIEntID + "=swarm:all:allow"
```

Comment:
`APIEntAccessID` ? (access can be either allow or deny, people might think this only whitelists)

Reply:
So, at present, the value (allow/deny) is processed by `NewStringEntitlement` (through `parser.ParseStringEntitlement`). this means ha we'll need to have two different API IDs for `alllow` and `deny`, so the access specified is no lost in translation. 

Since this is the first Entitlement to have a value, we might be able  to account for the `value` processing elsewhere perhaps?